2022/src/day07.scala

```diff
@@ -0,0 +1,95 @@
+package day07
+
+import locations.Directory.currentDir
+import inputs.Input.loadFileSync
+
+@main def part1: Unit =
+  val root: Directory = Directory("/")
+  run(loadInput(), List(root))
+  val solution =
+    allSubdirs(root)
+      .map(totalSize)
+      .filter(_ <= 100_000L)
+      .sum
+  println(s"The solution is $solution")
+
+@main def part2: Unit =
```

Comment:
to integrate with the article which has a solver widget, would it be possible to delegate part1 and part2 to two extra functions that take a string parameter (the loaded input)